src/overlays/actors/ovl_En_Pamera/z_en_pamera.h

```diff
@@ -5,9 +5,35 @@
 
 struct EnPamera;
 
+typedef void (*EnPameraActionFunc)(struct EnPamera*, GlobalContext*);
+typedef void (*EnPameraSetupFunc)(struct EnPamera*, GlobalContext*);
+
 typedef struct EnPamera {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x1E4];
+    /* 0x144 */ ColliderCylinder collider;
+    /* 0x190 */ SkelAnime skelAnime;
+    /* 0x1D4 */ EnPameraActionFunc actionFunc;
+    /* 0x1D8 */ EnPameraSetupFunc setupFunc;
+    /* 0x1DC */ Vec3s* pathPoints;
+    /* 0x1E0 */ s32 pathIndex;
+    /* 0x1E4 */ s32 pathPointsCount;
+    /* 0x1E8 */ AnimatedMaterial* animatedTextures; 
```

Comment:
This doesn't seem like this gets used an an AnimatedMaterial. Am I missing something?

Reply:
You are right, I must have missed this when going through cleaning up, I originally thought it was that, but it is just part of the path.